assessment/services/test_flow.py

```diff
@@ -0,0 +1,63 @@
+from django.contrib.auth.models import User
+from one_day_intern import utils as odi_utils
+from one_day_intern.exceptions import InvalidRequestException
+from . import utils
+from ..exceptions.exceptions import AssessmentToolDoesNotExist
+from ..models import TestFlow
+
+
+def validate_test_flow_registration(request_data: dict):
+    if not odi_utils.text_value_is_valid(request_data.get('name'), max_length=50):
+        raise InvalidRequestException('Test Flow name must exist and must be at most 50 characters')
+
+    if request_data.get('tools_used') and not isinstance(request_data.get('tools_used'), list):
+        raise InvalidRequestException('Test Flow must be of type list')
+
+    if request_data.get('tools_used'):
+        tools_used = request_data.get('tools_used')
+        for tool_used_data in tools_used:
+            tool_id = tool_used_data['tool_id']
+            tool_release_time = tool_used_data['release_time']
+
+            try:
+                utils.get_tool_from_id(tool_id)
```

Comment:
Hello, Jo. I see you have got the tool based on the ID, but if I may ask, can we actually verify that the assessment tool is made by the company?

Reply:
Hello, Alex. Nice catch! I am actually planning to implement this in the next commit. However, this commit has served the basic functionality to create a test flow. Do you think its better if we can merge first so that you guys in the Front End can directly integrate to the backend?

Reply:
Aight Jo, I'll merge it now. We want to integrate them as quickly as possible to get some deliverables out. Please do it in your next commit, Jo.